Sources/BitcoinKit/Core/Network.swift

```diff
@@ -25,28 +25,42 @@
 
 import Foundation
 
-public class Network {
+open class Network {
```

Comment:
maybe `Network` can be merged together with new `CoinType`, being a protocol and "open enum" (`struct`, **not** an `enum`) to ease of use for other DLTs :)

Reply:
I look into Network, and now I realize that it definitely needs more refactorization.
But I want to leave it for now. It's going to be another PR.